sqlmesh/dbt/target.py

```diff
@@ -255,3 +265,61 @@ def to_sqlmesh(self) -> ConnectionConfig:
             access_token=self.token,
             concurrent_tasks=self.threads,
         )
+
+
+class BigQueryConfig(TargetConfig):
+    """
+    Project connection and operational configuration for the BigQuery target
+
+    Args:
+        schema_: Overrides
+        type: The type of the target (bigquery)
+        method: The BigQuery authentication method to use
+        project: The BigQuery project to connect to
+        location: The BigQuery location to connect to
+        keyfile: The path to the BigQuery keyfile
+        keyfile_json: The BigQuery keyfile as a JSON string
+        token: The BigQuery token
+        refresh_token: The BigQuery refresh token
+        client_id: The BigQuery client ID
+        client_secret: The BigQuery client secret
+        token_uri: The BigQuery token URI
+        scopes: The BigQuery scopes
+    """
+
+    schema_: str = Field(alias="dataset")
```

Comment:
schema is defined in the base class. Did you mean to overwrite it here? I believe we'll still need to alias "schema" also.

Reply:
Yeah I didn't realize dbt supported both `schema` and `dataset` for BigQuery. Updated: https://github.com/TobikoData/sqlmesh/pull/648/commits/cbc003bec427fc35e7c0caa68bb87d7ccabcc39a